core/src/components/select/select.ios.scss

```diff
@@ -21,11 +21,11 @@
   color: #{$text-color-step-350};
```

Comment:
Optional:

It's a bit odd that the icons can move around as you select content. I would have expected it to behave more like the floating labels. That being said, the start/end/fixed labels aren't part of the MD/iOS spec so we have some freedom with how they behave.

Example:

https://github.com/ionic-team/ionic-framework/assets/2721089/bb0a3b8f-c6f1-4a6f-9bd7-8c946622ead7

Reply:
I'll leave this as-is for now since the toggle icon behaves in the same way on `main`, moving around as the selected item changes. We can always update this if people report it as a bug.